test/tools/libtest.sh

```diff
@@ -228,19 +228,29 @@ ORIG_IFS=
 assert_equals()
 {
 	file="$1"; shift
+	whitespace_arg="ignore";
```

Comment:
This default value threw me off while reviewing. Might have been clearer if it defaulted to `-w` and the value parsing was inside the `if [ "$#" -gt 1 ]`.